react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -153,9 +166,11 @@ function _mapStateToProps(state, ownProps) {
         = getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.AUDIO, id);
     const videoTrack
         = getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.VIDEO, id);
+    const { visibleParticipantIds } = state['features/filmstrip'];
 
     return {
         _audioTrack: audioTrack,
+        _hideVideo: !visibleParticipantIds.includes(id),
```

Comment:
IMHO this prop should come as an own prop instead of through a redux connection because if / when we work on a tile view, we'll probably want to reuse the Thumbnail component, but it might not be rendered in a FlatList.

Reply:
I'm more in favour with self-containing components, and it would be weird for me to get this from the outside when we have the same information (Redux) in the inside. You're right however that if we reuse this, then we may need this externally, but then there would be a lot of changes needed (e.g. relocating the entire component from the Filmstrip feature) so I think I'd go for the logical solution here other than the future proof one.

Reply:
We already know we are going to implement a tile view. So why do the work twice? Let's advance in the general direction we intend to follow instead...

Reply:
Will that be the only available view?